app/Http/Controllers/Admin/NodesController.php

```diff
@@ -28,6 +28,7 @@
 use Log;
 use Alert;
 use Validator;
+use Carbon\Carbon;
```

Comment:
You can access Carbon with `use Carbon;` due to [this](https://github.com/Pterodactyl/Panel/blob/develop/config/app.php#L185).